docs/content_tabs.rst

```diff
@@ -100,7 +89,80 @@ the sphinx-immaterial theme provides its own directives to make use of content t
                         :start-after: /* *********************** custom-tab-item-style
                         :end-before: /* ************************* inline icon stuff
 
+.. _linked_tabs:
+
+Linked tabs
+-----------
+
+Content tabs that share the same label can be selected synchronously by adding
+:python:`"content.tabs.link"` to the list of :themeconf:`features` in conf.py.
+
+.. code-block:: python
+    :caption: in conf.py
+
+    html_theme_options = {
+        "features": [
+            "content.tabs.link",
+        ],
+
+Include :python:`"navigation.instant"` in the list of :themeconf:`features` to have the
```

Comment:
`navigation.instant` shouldn't be needed if this is working properly; if it doesn't, that suggests a problem with the use of localStorage --- also in my experience `navigation.instant` is kind of buggy and doesn't provide any significant benefit so I'm not sure we want to recommend it

Reply:
I quick copied and pasted the linked tabs example into a separate page and verified it worked with the feature specified in conf.py. I didn't think to try without it since the mkdocs-material doc mandated it be specified. I'll re-check without it explicitly enabled.

Reply:
You're correct, `navigation.instant` isn't required for cross-page syncing of linked tabs. I have adjusted the doc here to express this difference from upstream.

Going forward, we should probably be careful about tab names. For example, a tab named "CSS hint" would be shown on page load if another tab of the same name was the last selected tab from a different section/page.

Reply:
Agreed.  We also might wish to improve on the functionality provided by mkdocs-material to allow linking tabs more selectively, rather than an all or nothing option.  I also think in some cases it is used as an example for this theme in a way that might not be ideal for actual use.  My thinking is that tabs are best when there are true alternatives, where the user is unlikely to be interested in both options.  E.g. installation instructions for windows vs linux, or usage examples in Python vs C++.  If the user is likely to care about multiple tabs, then it may be better not to use tabs and instead just use sections.

Reply:
It is true that I tend to use tabs as a way of demonstrating steps in an example. My intention there was to keep the page length from growing to an overwhelming size -- I have a bad habit of using my mouse wheel more than the toc entries' links.

Maybe we should open an issue/discussion upstream about specifying certain tabs sets as non-sync-able?